deployment/ccip/changeset/cs_deploy_token_pools.go

```diff
@@ -121,8 +123,14 @@ func (c DeployTokenPoolContractsConfig) Validate(env deployment.Environment) err
 		if !ok {
 			return fmt.Errorf("chain with selector %d does not exist in state", chainSelector)
 		}
-		if router := chainState.Router; router == nil {
-			return fmt.Errorf("missing router on %s", chain.String())
+		if c.UseTestRouter {
```

Comment:
nit : following the same name what we have for other changeset, feel free to ignore.
```suggestion
		if c.IsTestRouter {
```

Reply:
https://github.com/smartcontractkit/chainlink/pull/16178/commits/07f41ae148929c69cf98e26d5f2e23de9618494a